.github/workflows/test.yml

```diff
@@ -51,32 +51,28 @@ jobs:
         run: make test
       - name: Upload coverage reports to Codecov
         uses: codecov/codecov-action@v3
-      - name: Build bianry for E2E testing
+      - name: Build Linux Binary
         run: |
-          ./scripts/build.sh v0.0.1 linux-x64
-          ./scripts/build.sh v0.0.1 win-x64
-          ./scripts/build.sh v0.0.1 osx-x64
+          # the binary will be used in E2E test
+          python3 ./scripts/build.py v0.0.1 linux-x64 --enable-aot
```

Comment:
Question: I notice that this command is building binary v0.0.1, is it building the binary of notation-akv plugin? If yes, should we test the most recent version, i.e., v1.0.2?

Reply:
This is a temporary version for building the code based on the current commit. We want to run the E2E tests based on the current code rather than the recent release.